Sources/RoktUXHelper/Data/Expansion/Domain/BNFCatalogMapping.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  BNFCatalogMapping.swift
+//  RoktUXHelper
+//
+//  Licensed under the Rokt Software Development Kit (SDK) Terms of Use
+//  Version 2.0 (the "License");
+//
+//  You may not use this file except in compliance with the License.
+//
+//  You may obtain a copy of the License at https://rokt.com/sdk-license-2-0/
+
+import Foundation
+
+@available(iOS 15, *)
+struct BNFCatalogMapping<DE: DataExtractor>: BNFMapper where DE.U == CatalogItem {
+
+    private let extractor: DE
```

Comment:
Is it possible to avoid a short acronym like `DE` in all of these class types? It makes the code harder to read as you're having to mentally map what `DE` actually refers to. Additionally makes code searching and similar harder

A great example is of this is `DE.U` (and I know it's a further refactor) where a developer will have to go to another class just to understand this class

Reply:
good point, just used to using really short names for generics. also one less name to think about 😅